int8 responder_id
int8 responder_idx
float64 distance
uint8[] data
uint32 uwb_time
geometry_msgs/Point responder_location
